index.css

```diff
@@ -7,59 +7,82 @@ DARK: #44354A
 COLOR: #9E4770
 */
 
+
+
 html, body {
     margin: 0;
     padding: 0;
     box-sizing: border-box;
-    font-size: 10px;
     background: #201A23;
-    font-family: 'Poppins';
+    font-family: 'Poppins', sans-serif; 
     color: #FBFBFB;
 } 
 
 .salesboard {
-    position: fixed;
-    top: 0;
-    bottom: 0;
-    left: 0;
-    right: 0;
-    margin: auto;
-    text-align: center;
-    width: 50vw;
-
-    display: flex;
-    flex-direction: column;
-    align-items: center;
+   text-align: center;
+   max-width: 940px;
+   margin: 0 auto;
+   padding: 0 15rem;
```

Comment:
This padding only has effect when the container is less than 940px; when you set this to 2 or 3 rem it will show correct on small devices. That way we're responsive ready :)
Some bottom padding on the last bars with the revenue and commission will prevent them sticking to the bottom of the browserwindow